bchain/coins/eth/ethaddrcontracts.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option go_package = "github.com/trezor/blockbook/bchain/coins/eth";
+
+message ProtoAddrContracts {
```

Comment:
update addr contracts to proto type to fix performance impact of manual pack/unpack related to huge multi addr contracts on chain

Reply:
seemed to help a bit, but sync has since crawled to a stop...